magenta-lib/src/main/scala/magenta/deployment_type/Lambda.scala

```diff
@@ -16,6 +16,9 @@ object Lambda extends DeploymentType  {
       |
       |It is recommended to use the `bucket` parameter as storing the function code in S3 works much better when using
       |cloudformation.
+      |
+      |Ensure to add any relevant dependencies to your deploy in your riff-raff.yaml to guard against race conditions.
```

Comment:
Maybe add something like `(such as a Riff-Raff Cloudformation update modifying your lambda at the same time this deployment type is running)` to more explicitly outline the potential pitfall.